### DISCUSSION TITLE
changing measurement length color

### DISCUSSION THREAD

**Discussion by Avinash-c-gowda** [nroduit/Weasis#453] [General]
Hi @nroduit , is that possible to change the measurement length color;  currently its taking by default white color

**Comment by nroduit** [comment]
No, the default color should be yellow.
You can change the default color in the drawing pane (the current version has [a bug with the preference persistence](https://github.com/nroduit/Weasis/issues/435))

![image](https://github.com/nroduit/Weasis/assets/993975/94c281b7-e3f6-4bba-9eaf-a6d1565ae00a)

**Comment by Avinash-c-gowda** [comment]
ya , but iam talking about this (Length : 168.8 pix) view default color, while drawing it is showing length of the line rt how can i change that color

**Comment by Avinash-c-gowda** [comment]
@nroduit  could u please update on this comment

**Comment by nroduit** [comment]
No, there's no option to change the text color. This is to ensure text visibility on any background.

**Reply by Avinash-c-gowda** [reply]
ok, thanks for the reply
